packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
   * Migrate docs from terra-layout
   * Added docs for `terra-filter-pills`
   * Added tests for `terra-filter-pills`
+  * Migrate docs from terra-theme-provider
```

Comment:
This needs to be move up to the Unreleased section. The entry for terra-menu needs to move also.

Reply:
changed here a1afa77765067ecdee1efaa35475d5da61c94495